ionicons/IonIcons.m

```diff
@@ -19,10 +19,33 @@ @implementation IonIcons
 + (UIFont*)fontWithSize:(CGFloat)size;
 {
     UIFont* font = [UIFont fontWithName:@"ionicons" size:size];
+    if (!font) {
+        [[self class] dynamicallyLoadFontNamed:@"ionicons"];
```

Comment:
Cool happy to help. Thanks for the great lib, let me know if you'd like me to check in any more changes so you can merge the PR.